### ISSUE TITLE
Crash when changing landscape/portrait mode

### ISSUE THREAD

**Issue by heinhuiz** [stefan-niedermann/nextcloud-notes#919]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Describe the bug**

App crashes when editing a note and changing from portrait to landscape mode.

**To Reproduce**
1.  Create a note or open an existing note in editing mode
2. Change screen orientation from portrait to landscape and/or v.v.
3. Often the app crashes to the crash screen, often closes showing the crash screen at reopening, sometimes crashes with popup "Notes keep crashing"

**Expected behavior**

Change from portrait to landscape or v.v.

**Screenshots**

**Smartphone (please complete the following information):**
 - Nextcloud Notes-Version (android app): 2.16
4
 - F-Droid or Play Store: F-droid
 - Android-Version: 10
 - Device: Samsung Galaxy A70/SM-A705FN/DS

**Server**
 - Nextcloud version: 
 - Nextcloud Notes version (server app): 

**Stacktrace**
```
App Version: 2.16.4
App Version Code: 2016004
App Flavor: fdroid

Files App Version Code: 30120190

---

OS Version: 4.14.117-17976949(A705FNXXU5BTF1)
OS API Level: 29
Device: a70q
Manufacturer: samsung
Model (and Product): SM-A705FN (a70qxx)

---

java.lang.RuntimeException: Unable to start activity ComponentInfo{it.niedermann.owncloud.notes/it.niedermann.owncloud.notes.edit.EditNoteActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.view.ViewGroup.removeAllViews()' on a null object reference
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3448)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3595)
	at android.app.ActivityThread.handleRelaunchActivityInner(ActivityThread.java:5536)
	at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:5444)
	at android.app.servertransaction.ActivityRelaunchItem.execute(ActivityRelaunchItem.java:69)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2147)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:237)
	at android.app.ActivityThread.main(ActivityThread.java:7814)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1068)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.view.ViewGroup.removeAllViews()' on a null object reference
	at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:545)
	at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:166)
	at it.niedermann.owncloud.notes.edit.EditNoteActivity.onCreate(EditNoteActivity.java:54)
	at android.app.Activity.performCreate(Activity.java:7955)
	at android.app.Activity.performCreate(Activity.java:7944)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1307)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3423)
	... 13 more

```

**Comment by stefan-niedermann** [comment]
Thank you for the report. Will have a look as soon as possible.

**Comment by stefan-niedermann** [comment]
@heinhuiz @MickaelBergem @Hunter16g

I think i found a solution and would like to ask you to test [this `apk`](https://github.com/stefan-niedermann/nextcloud-notes/suites/956271662/artifacts/11978601) which contains the fix.

After unzipping it can simply be installed in *parallel* to the versions from F-Droid as well as Play Store.

Thank you for the report and thanks for helping getting this fixed :slightly_smiling_face:

**Comment by Hunter16g** [comment]
Fixes it for me :)

**Comment by heinhuiz** [comment]
Hi Stefan, thank you very much for the quick response and solution! It fixes the crash for me too, but I found some other strange behaviour of which I am not sure whether it is introduced now or existed before.

With the dark theme selected, and the screen rotated to landscape, the background changes from black to white and a button 'Done' appears. That does not happen in portrait mode.

![Screenshot_20200723-173118_Nextcloud Notes](https://user-images.githubusercontent.com/12214714/88319043-21957a00-cd0b-11ea-86bd-48e4c391d3c5.jpg)

As said, I am not sure if this is a result of the fix or preexisting already.

**Comment by stefan-niedermann** [comment]
@heinhuiz this is a default android behavior for multiline textareas - i can't do anything to prevent android from this behavior AFAIK. And this behavior has been there all time long. Thank you though for notfying me.

@Hunter16g and all the others: Thank you for testing. This fix will be shipped in `v2.16.5`.

Cheers
